### PR TITLE
Revert to previous version of deleted entities (#986)

### PR DESCRIPTION
**Pull request by jessebrennan**

### CODE REVIEW

File: test/indexer/test_hca_indexer.py [hunk 1]
```diff
@@ -246,12 +245,17 @@ def test_bundle_downgrade(self):
         self._assert_old_bundle(num_expected_new_contributions=5, ignore_aggregates=True)
         self._assert_new_bundle(num_expected_old_contributions=5)
 
-    def _assert_old_bundle(self, num_expected_new_contributions=0, ignore_aggregates=False, ignore_deletes=False):
+    def _assert_old_bundle(self,
+                           num_expected_new_contributions=0,
+                           num_expected_new_deleted_contributions=0,
+                           ignore_aggregates=False,
+                           ignore_deletes=False):
```
Comment (hannes-ucsc) [inline test/indexer/test_hca_indexer.py]: I have the feeling this parameter is a) badly named (shouldn't it be called `expect_deletions`?) and b) made redundant by `num_expected_new_deleted_contributions`.

**Comment by hannes-ucsc** [comment]
Rebased on develop and force-pushed.

File: test/indexer/test_hca_indexer.py (hunk 1 above)
Comment (jessebrennan) [reply test/indexer/test_hca_indexer.py]: You're right, it is redundant. I removed it.

File: test/indexer/test_hca_indexer.py [hunk 2]
```diff
@@ -38,7 +37,7 @@ def setUpModule():
 
 class TestHCAIndexer(IndexerTestCase):
 
-    def _get_hits(self):
+    def _get_all_hits(self):
```
[the remark this answers is not in the document]
Comment (jessebrennan) [reply test/indexer/test_hca_indexer.py]: done.

**Pull request merged**